lib/template/spec/spec_helper.rb

```diff
@@ -5,7 +5,7 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 #
-ENV["RACK_ENV"] = "test"
+ENV["PLINY_ENV"] = "test"
```

Comment:
@dmathieu Hahaha, nice! I suspect that this might be too much of a breaking change for them at this point though (too many test suites out there relying on it), but I hope you land it.